content/post/practical-stegen.Rmd

```diff
@@ -809,15 +811,25 @@ ggplot(rr_df, aes(x = values, y = ind)) +
 The results are a lot clearer now: the tiramisu is by far the largest risk
 factor in this outbreak.
 
+# Plotting a very basic spatial overview of cases
```

Comment:
@AmmerB, it's good practice to insert vertical spaces between different markdown elements (in this case: header and body).